src/ray/common/file_system_monitor.h

```diff
@@ -63,11 +63,14 @@ class FileSystemMonitor {
   bool OverCapacityImpl(const std::string &path,
                         const std::optional<std::filesystem::space_info> &info) const;
 
+  void InitializeInitialCapacity(const std::string &path);
+
  private:
   FRIEND_TEST(FileSystemTest, TestOverCapacity);
   const std::vector<std::string> paths_;
   const double capacity_threshold_;
   std::atomic<bool> over_capacity_;
+  std::atomic<uint64_t> initial_capacity_{0};
```

Comment:
is this used anywhere?

Reply:
oh. Sorry about that, I just fixed. It's some config that I used to testing. Sorry! Now it fixed now! Thanks!